gateway/src/services/config-manager-v2.ts

```diff
@@ -160,9 +160,19 @@ export class ConfigurationNamespace {
       ) as Configuration;
       deepCopy(configCandidate, configTemplateCandidate);
       if (!this.#validator(configTemplateCandidate)) {
-        throw new Error(
-          `Configuration for namespace ${this.id} seems to be outdated/broken. Kindly fix manually.`
-        );
+        for (const err of this.#validator.errors as DefinedError[]) {
+          switch (err.keyword) {
+            case 'additionalProperties':
+              throw new Error(
+                `${this.id} config file seems to be outdated/broken due to additional property "${err.params.additionalProperty}". Kindly fix manually.`
+              );
+              break;
+            default:
+              throw new Error(
+                `${this.id} config file seems to be outdated/broken due to "${err.keyword}" - ${err.message}. Kindly fix manually.`
+              );
```

Comment:
I believe each case block must end with a `break;` keyword

Reply:
I would just change it to an if....else.
The reason I didn't include initially was because typescript complained the break statement would be unreachable.